Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
> Is it stable?

Relatively. We disabled OpenSSL and flipped the switch on SecureTransport when it became available and so far it's been pretty smooth. We've had to do [some](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/qt5.rb#L47-L55) patching of a few issues with QtWebKit but otherwise no reports of issues so far.